releasenotes/notes/slsqp-optimizer-199a4694e480297e.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
```

Comment:
Can we update this to include the fact that CobylaOptimizer was enhanced with the multi-start (trials) capability.

Reply:
Done